src/features/content-explorer/content-explorer-modal-container/ContentExplorerModalContainer.js

```diff
@@ -37,6 +37,13 @@ class ContentExplorerModalContainer extends Component {
         /** Folder is in the process of being created. */
         onSelectedClick: PropTypes.func,
         /** Called when selected button is clicked */
+        onSelectItem: PropTypes.func,
+        /**
```

Comment:
Looks like this comment should be above the prop. Looks like some other pre-existing comments are also misplaced. I think this happens when auto-formatting re-orders the props alphabetically but it doesn't move the comments

Reply:
Thanks for catching this! Updated.